### PR TITLE
Font picker consistency

### PR DESCRIPTION
**Pull request by xurble**
This will move the option for SF Rounded from its own toggle into the popup menu with Hyperlegible and Dyslexia.

### CODE REVIEW

**Comment by Dimillian** [comment]
You probably need to get main :p

**Comment by Dimillian** [comment]
Raise a bug, get a commit lol

File: Packages/Env/Sources/Env/UserPreferences.swift [hunk 1]
```diff
@@ -156,3 +153,16 @@ public class UserPreferences: ObservableObject {
     recentlyUsedLanguages = Array(copy.prefix(3))
   }
 }
+
+
+
+// https://stackoverflow.com/a/67305343
+extension UIFont {
```
Comment (Dimillian) [inline Packages/Env/Sources/Env/UserPreferences.swift:160]: Does it needs to be in two places?

File: Packages/DesignSystem/Sources/DesignSystem/Font.swift [hunk 2]
```diff
@@ -66,3 +73,14 @@ public extension Font {
     customFont(size: userScaledFontSize(baseSize: caption), relativeTo: .caption)
   }
 }
+
+
+
+extension UIFont {
```
Comment (Dimillian) [inline Packages/DesignSystem/Sources/DesignSystem/Font.swift:79]: Here too

Comment (xurble) [reply Packages/DesignSystem/Sources/DesignSystem/Font.swift:79]: No. That's weird. Let me sort it along with the merge conflict.

**Comment by xurble** [comment]
This should be in proper shape now.

**Comment by Dimillian** [comment]
Indeed much better

**Pull request merged**